quandl/model/datatable.py

```diff
@@ -14,3 +30,54 @@ def get_path(cls):
     def data(self, **options):
         updated_options = Util.convert_options(**options)
         return Data.page(self, **updated_options)
+
+    def bulk_download_file(self, file_or_folder_path, **options):
+        if not isinstance(file_or_folder_path, str):
+            raise QuandlError(Message.ERROR_FOLDER_ISSUE)
+
+        return self._url_request(file_or_folder_path, **options)
+
+    def _url_request(self, file_or_folder_path, **options):
+        url = self._download_request_path()
+        code_name = self.code
+        if 'params' not in options:
+            options['params'] = {}
+        if ApiConfig.api_key:
+            options['params']['api_key'] = ApiConfig.api_key
+        if ApiConfig.api_version:
+            options['params']['api_version'] = ApiConfig.api_version
+
+        if list(options.keys()):
```

Comment:
Did you mean `if options['params']:`?

Reply:
I updated this part since this api doesn't need to support more parameters now